package.json

```diff
@@ -65,6 +65,7 @@
     "is-plain-object": "^3.0.0",
     "js-yaml": "^3.13.1",
     "keycloak-connect": "^7.0.1",
+    "moment": "^2.24.0",
```

Comment:
@johnworth - We already have date formatting lib in `ui-lib`. https://github.com/cyverse-de/ui-lib/blob/master/src/util/DateFormatter.js. We are using `date-fns` since `moment` mutation making it buggy.